bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script-pre-k8s-1-31.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
2024
(because it's a new file, even if we are just copying code to it)

Reply:
github is misdisplaying it a bit. This is the old file, and just the same :-)

But thanks for the pointer, have to bump the year on the other file!

Reply:
> github is misdisplaying it a bit. 

hm, not sure how so.

seems like bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script-pre-k8s-1-31.sh is a new file that has the old contents. it should be with the 2024 year. old file can have the new changes but be with the old year.

Reply:
It's basically a `git mv bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script-pre-k8s-1-31.sh` 🤔

Reply:
FWIW, it ends up as a new file in the tree, so the correct action is to apply 2024.
i don't think the license police will mind, so up to you.

Reply:
There is a license police? :)